python/lsst/ts/observatory/control/auxtel/atcalsys.py

```diff
@@ -196,11 +196,38 @@ async def prepare_for_flat(self, config_name: str) -> None:
             else utils.make_done_future()
         )
 
+        task_setup_electrometer = self.setup_electrometers(
+            mode=int(config_data["electrometer_mode"]),
+            range=float(config_data["electrometer_range"]),
+            integration_time=float(config_data["electrometer_integration_time"]),
+        )
+
         await asyncio.gather(
-            task_setup_monochromator,
-            task_setup_latiss,
+            task_setup_monochromator, task_setup_latiss, task_setup_electrometer
         )
 
+    async def setup_electrometers(
+        self, mode: int = 1, range: float = -1, integration_time: float = 0.1
+    ) -> None:
+        """Setup all electrometers."""
+        for electrometer in [
+            getattr(self.rem, component_name)
+            for component_name in self.components_attr
+            if "electrometer" in component_name
+        ]:
+            await electrometer.cmd_setMode.set_start(mode=mode)
+            await electrometer.cmd_setRange.set_start(setRange=range)
+            await electrometer.cmd_setIntegrationTime.set_start(
+                intTime=integration_time
+            )
+            await electrometer.cmd_performZeroCalib.start(timeout=self.long_timeout)
```

Comment:
Should this be `set_start` or just `start`? @tribeiro

Reply:
just start.. we only use `set_start` when we need to pass command parameters. Note that timeout is not a command parameter.